machine/v1beta1/types_vsphereprovider.go

```diff
@@ -190,6 +190,9 @@ type Workspace struct {
 	// ResourcePool is the resource pool in which VMs are created/located.
 	// +optional
 	ResourcePool string `gcfg:"resourcepool-path,omitempty" json:"resourcePool,omitempty"`
+	// VMGroup is the cluster vm group in which virtual machines will be added for vm host group based zonal.
+	// +optional
+	VMGroup string `gcfg:"vmGroup,omitempty" json:"vmGroup,omitempty"`
```

Comment:
How does this interact with the infrastructure changes?

What is the implementation of this in CAPZ, how will we migrate this?

Reply:
Its fine if you would rather see it as another pr. The machine needs the vmgroup for machine-api-operator to know which group to add a virtual machine into.

https://github.com/openshift/machine-api-operator/pull/1285/commits/50aa0d4b5357b3bda3e2738bfb6c60793c7c7dc1#diff-21a982202fdcdf4beffc4d00659798b54717940bf07f685006dbcb99d9dbf0ecR1143

Reply:
CAPV vm-host zonal is already being used in the installer for this effort.

https://github.com/openshift/installer/pull/8873/commits/24d2fad365927f4f9728aed3acf0f018ea736b6f#diff-4f0946cfde9ce4a09f7a7fc9fb4a0b6b162f216e1ddc6ccbbfcac0965ce7e687R116-R119